system/jlib/jlog.cpp

```diff
@@ -2324,6 +2331,53 @@ MODULE_EXIT()
     thePassAllFilter = NULL;
 }
 
+void updateContainerizedLogMsgHandlerDetail(LogMsgDetail logDetail)
+{
+    updateContainerizedLogMsgHandler(getCategoryLogMsgFilter(MSGAUD_all, MSGCLS_all, logDetail));
+}
+
+void updateContainerizedLogMsgHandlerLogFields(unsigned fields)
+{
+	theStderrHandler->setMessageFields(fields);
+}
```

Comment:
don't think you need this.
You can call this when installing the log handler with what you pick up from logging/@logFields, or default if empty.